django_project/frontend/src/containers/MainPage/Metrics/index.tsx

```diff
@@ -94,10 +130,27 @@ const Metrics = () => {
     const handleEmptyPopulationEstimateCategoryCountPercentage = (isEmpty: boolean | ((prevState: boolean) => boolean)) => {
         setHasEmptyhasEmptyPopulationEstimateCategoryCountPercentage(isEmpty);
     };
-    const handleHasEmptyAreaAvailable = (isEmpty: boolean | ((prevState: boolean) => boolean)) => {
+    const handleEmptyAreaAvailable = (isEmpty: boolean | ((prevState: boolean) => boolean)) => {
         setHasEmptyAreaAvailable(isEmpty);
     };
 
+    const imageUrl = 'http://localhost:8000/static/images/icons/download.svg';
+
+    // Get the current window location (i.e., the full URL)
+    let currentLocation = window.location.href;
+
+    // Remove '/charts' from the current URL
+    currentLocation = currentLocation.replace('/charts', '');
+
+    // Replace 'localhost:8000' with the current full URL
+    const imageUrlWithDomain = imageUrl.replace('http://localhost:8000', currentLocation);
+
+    const informationIconUrl = 'http://localhost:8000/static/images/icons/information.svg';
+
+    // Replace 'localhost:8000' with the current full URL
+    const informationIconUrlWithDomain = informationIconUrl.replace('http://localhost:8000', currentLocation);
```

Comment:
@tinashechiraya can you just use `/static/...` url? Without the domain.

Reply:
@dimasciput I have resolved this